### PR TITLE
Milestone 2 components

### PR DESCRIPTION
**Pull request by KerronKing**
In this milestone I:

- Created an App component to render the Display and ButtonPanel components.
- Created a Display component that accepts the incoming result as a string.
- Created a Button component that accepts a name prop as a string.
- Created a ButtonPanel component that renders the buttons in a calculator panel format.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import ButtonPanel from './button-panel';
+
+const App = () => (
+  <div>
+    <Display />
+    <ButtonPanel />
+  </div>
+);
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (shloch) [inline src/app.js]: - this App component should be in folder `src/components`

File: src/display.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+Display.defaultProps = {
+  result: 0,
+};
+
+export default Display;
```
Comment (shloch) [inline src/display.js]: - this component should be in folder `src/components`

File: src/button.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <div>{name}</div>
+  );
+};
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+};
+
+export default Button;
```
Comment (shloch) [inline src/button.js]: - this component should be in folder `src/components`

File: src/button-panel.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './button';
+
+const ButtonPanel = () => (
+  <div>
```
Comment (shloch) [inline src/button-panel.js]: - this component should be in folder `src/components`

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @KerronKing 

Nice work, nothing to report, apart of file organisation as prescribed by project description 😊😊

Please just correct this and get a thumbs up.. (waiting for last re-review)

Happy coding

**Comment by KerronKing** [comment]
All files moved to components folder as per instructions.

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED

Hi @KerronKing this looks great now, thanks for making changes, please merge and move on to the next milestone :+1:   **This is approved** :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**